$OpenBSD: patch-CMakeLists_txt,v 1.2 2016/08/11 11:29:10 jasper Exp $

ninja: error: build.ninja:111: bad $-escape (literal $ must be written as $$)

--- CMakeLists.txt.orig	Wed Jul 27 21:54:33 2016
+++ CMakeLists.txt	Thu Aug 11 13:25:49 2016
@@ -47,7 +47,7 @@ add_custom_target(${quest_name}_data
 add_custom_command(
   OUTPUT ${quest_name}
   COMMAND echo '\#!/bin/sh' > ${quest_name}
-  COMMAND echo 'solarus-run ${CMAKE_INSTALL_PREFIX}/share/solarus/${quest_name} $*' >> ${quest_name}
+  COMMAND echo 'solarus-run ${CMAKE_INSTALL_PREFIX}/share/solarus/${quest_name} $$*' >> ${quest_name}
 )
 add_custom_target(${quest_name}_command
   ALL
